.asf.yaml

```diff
@@ -34,3 +34,4 @@ github:
     projects: false
   autolink_jira:
     - CASSANDRA
+    - CASSGO
```

Comment:
Good to keep CASSANDRA, quite likely there'll be an occasional ticket that cross between jira projects 👍